hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/service/ContractCallEvmCodesTest.java

```diff
@@ -16,347 +16,299 @@
+                        new BigInteger(
+                                "9918110051302171585080402603319702774565515993150576347155970296011118125764")));
     }
 
     @Test
-    void AltBN128MulPrecompiledContract() {
-        final var functionHash = functionEncodeDecoder.functionHashFor("multiplyPoints", EVM_CODES_ABI_PATH);
-        final var serviceParameters = serviceParametersForEvmCodes(functionHash);
-
-        assertThat(contractCallService.processCall(serviceParameters))
-                .isEqualTo(
-                        "0x0769bf9ac56bea3ff40232bcb1b6bd159315d84715b8e679f2d355961915abf02ab799bee0489429554fdb7c8d086475319e63b40b9c5b57cdf1ff3dd9fe2261");
+    void AltBN128MulPrecompiledContract() throws Exception {
+        final var contract = testWeb3jService.deploy(EvmCodes::deploy);
+        var result = contract.call_multiplyPoints().send();
+        assertThat(result)
+                .isEqualTo(new G1Point(
+                        new BigInteger("3353031288059533942658390886683067124040920775575537747144343083137631628272"),
+                        new BigInteger(
+                                "19321533766552368860946552437480515441416830039777911637913418824951667761761")));
     }
 
     @Test
-    void AltBN128PairingPrecompiledContract() {
-        final var functionHash = functionEncodeDecoder.functionHashFor("pairingCheck", EVM_CODES_ABI_PATH);
-        final var serviceParameters = serviceParametersForEvmCodes(functionHash);
-
-        assertThat(contractCallService.processCall(serviceParameters)).isEqualTo(TRUE);
+    void AltBN128PairingPrecompiledContract() throws Exception {
+        final var contract = testWeb3jService.deploy(EvmCodes::deploy);
+        var result = contract.call_pairingCheck().send();
+        assertThat(result).isEqualTo(true);
     }
 
     @Test
-    void BLAKE2BFPrecompileContract() {
-        final var functionHash = functionEncodeDecoder.functionHashFor("blake2", EVM_CODES_ABI_PATH);
-        final var serviceParameters = serviceParametersForEvmCodes(functionHash);
-
-        assertThat(contractCallService.processCall(serviceParameters))
-                .isEqualTo(
-                        "0xba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5de4533cc9518d38aa8dbf1925ab92386edd4009923");
+    void BLAKE2BFPrecompileContract() throws Exception {
+        final var contract = testWeb3jService.deploy(EvmCodes::deploy);
+        var result = contract.call_blake2().send();
+        String expectedResultHexString =
+                "0xba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5de4533cc9518d38aa8dbf1925ab92386edd4009923";
+        List<byte[]> expetedList = convertBlake2HexToList(expectedResultHexString.substring(2));
+        assertThat(result.get(0)).isEqualTo(expetedList.get(0));
+        assertThat(result.get(1)).isEqualTo(expetedList.get(1));
```

Comment:
Something like this should simplify things, and then you can remove `convertBlake2HexToList`: 

```suggestion
final var contract = testWeb3jService.deploy(EvmCodes::deploy);
        List<byte[]> result = contract.call_blake2().send();
        var expectedResultHexString =
                "ba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5de4533cc9518d38aa8dbf1925ab92386edd4009923";
        var resultBytes = Bytes.concatenate(result.stream().map(Bytes::wrap).toArray(Bytes[]::new)).toArray();
        var expected = hexStringToByteArray(expectedResultHexString);
        assertArrayEquals(resultBytes, expected);
```

Reply:
Comment addressed. I applied the above suggestion.